packages/x-components/src/x-modules/search-box/components/search-input.vue

```diff
@@ -319,14 +319,16 @@ the message “enter” appears.
 
 _Type any term in the input field to try it out!_
 
-```vue
+```vue live
 <template>
-  <SearchInput
-    @UserPressedEnterKey="logUserPressedEnter"
-    @UserFocusedSearchBox="hasFocus = true"
-    @UserBlurredSearchBox="hasFocus = false"
-    @UserIsTypingAQuery="value = 'focus'"
-  />
+  <div>
+    <SearchInput
+      @UserPressedEnterKey="value = 'enter'"
```

Comment:
Why you remove    @UserBlurredSearchBox="hasFocus = false"?

Reply:
I did it because when pressing "Enter" `UserPressedEnterKey` and `UserBlurredSearchBox` were both called, so they were conflicting, but now that we are talking about this, i think that it's better to use value and hasFocus independently, so we can conserve the event and see them in real time